spaceship/spec/tunes/tunes_stubbing.rb

```diff
@@ -31,6 +31,8 @@ def itc_stub_login
         to_return(status: 200, body: "", headers: {})
 
       # Actual login
+      stub_request(:get, "https://idmsa.apple.com/appleauth/auth/signin?widgetKey=e0abc").
+        to_return(status: 200, body: '', headers: { 'x-apple-hc-bits' => "12", 'x-apple-hc-challenge' => "f8b58554b2f22960fc0dc99aea342276" })
```

Comment:
If you end up addressing @ainame 's comments above regarding the case of the headers, we'd need to update it here as well 👀